slick/src/sphinx/schemas.rst

```diff
@@ -78,8 +78,7 @@ object. The following ones are defined for ``JdbcProfile``:
 
 .. index:: null, nullable, NotNull
 
-  Slick automatically marked Optional column as nullable and non-Optional columnas 
-  non-nullable when creating the DDL statements for the table. 
+   Nullable columns are represented by Option[T]
```

Comment:
Hi @satendrakumar06 - slight mis-communication here.  This text ("Nullable columns...") already exists on line 36-37.  I suggest removing it from line 81, unless there's a good reason to duplicate it here.